binsync/decompilers/ghidra/client/src/main/java/binsync/BSGhidraServerAPI.java

```diff
@@ -420,6 +420,56 @@ public Boolean setGlobalVarName(String addr, String name) {
 		
 		return success;
 	}
+	
+	public Map<String, Object> getGlobalVariable(String addr)
+	{
+		var program = this.server.plugin.getCurrentProgram();
+		var symTab = program.getSymbolTable();
+
+		Map<String, Object> global_var = new HashMap<>();
+		for (Symbol sym: symTab.getAllSymbols(true)) {
+			if (sym.getSymbolType() != SymbolType.LABEL) {
+				continue;
+			}
+			
+			if (sym.getAddress().equals(this.strToAddr(addr))) {
```

Comment:
dont all the addresses need to be rebased? Like, the addr that is sent to Ghidra is always an offset. So when you get the addr you need to convert it to Ghidra form. Then when you are sending it back, you need to make sure the addr if an offset again. Unless you plan on doing this on the Python side.